src/Pyz/Yves/CatalogPage/Theme/default/components/molecules/filter-category/filter-category.scss

```diff
@@ -36,6 +40,38 @@
         }
     }
 
+    &__category-row {
+        flex-grow: 1;
+
+        &--no-children {
+            margin-left: rem(16);
+        }
+    }
+
+    &__category-target {
+        @include helper-effect-transition(max-height);
+        overflow: hidden;
+        max-height: rem(10000);
+        opacity: 1;
+
+        &--hidden {
+            max-height: 0;
+            opacity: 0;
+        }
```

Comment:
Can we use this hack, instead of animating 10x px?

https://css-tricks.com/transitioning-to-auto-height/

Reply:
Or with `grid`
https://keithjgrant.com/posts/2023/04/transitioning-to-height-auto/